### ISSUE TITLE
content: CSS :marker pseudo-element [2020-08-01]

### ISSUE THREAD

**Issue by kaycebasques** [GoogleChrome/web.dev#3326]
* draft: TODO
* [thread](https://groups.google.com/a/google.com/g/webdev-content/c/Vc1X_wz7c1s)

**Comment by argyleink** [comment]
rad, draft begun

cc @Loirooriol

**Comment by argyleink** [comment]
🏓  draft ready for review && link updated/added in the comment

**Comment by argyleink** [comment]
🏓  🙂

**Comment by argyleink** [comment]
`::marker` intent to ship https://www.chromestatus.com/feature/6710566854852608

we'd like to release this post with the feature in Chromium stable 👍

**Comment by argyleink** [comment]
can I PR this, or want me to wait for a review in the google doc?

**Comment by jpmedley** [comment]
Chrome 86 goes to beta on September 3. September 2 is the latest this should go live.

**Comment by kaycebasques** [comment]
Let's have @rachelandrew do a review in the Google Doc

**Comment by rachelandrew** [comment]
@kaycebasques @argyleink will do!

**Comment by kaycebasques** [comment]
@argyleink do you want another review from @rachelandrew or are you still updating the content?

**Comment by argyleink** [comment]
i updated the content more today, it's now ready for another review 👍  thanks!

**Comment by rachelandrew** [comment]
@argyleink did a bit of rearranging and added some comments.

**Comment by argyleink** [comment]
thank you @rachelandrew

Feels to me like this post is ready to move into a PR and out of Google Docs?

**Comment by rachelandrew** [comment]
@argyleink yeah - let me know when it's ready and I'll take a look for style guide consistency etc.

**Comment by argyleink** [comment]
ready! https://deploy-preview-3580--web-dev-staging.netlify.app/css-marker-pseudo-element/

i'll add a header image here in a bit

**Comment by rachelandrew** [comment]
cool - I'll have a look on Wednesday when I'm next doing Google things.
